README.md

```diff
@@ -98,18 +100,18 @@ and the standard YAML for a new exercise would look like this:
 ```
 ---
 layout: exercise
-title: Name of exercise
-subtitle: Short description of the exercise
-language: R
+title: Name_of_exercise
```

Comment:
I actually prefer the spaces here and on the next line because it makes clear that these values can include spaces.